app/views/foods/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<h2>Total number of items with sweet potatoes: <%= %></h2>
```

Comment:
This was a bit tricky to figure out. But, there was a key in the JSON response called `totalHits` that could have been passed from the facade >controller > view.